src/citrine/informatics/predictors.py

```diff
@@ -66,6 +67,28 @@ def get_type(cls, data) -> Type['Predictor']:
                 'Must be in {}.'.format(data['config']['type'], type_dict.keys())
             )
 
+    def _wrap_training_data(self, training_data: Optional[Union[DataSource, List[DataSource]]]) -> List[DataSource]:
```

Comment:
Could this be a static method?

Reply:
I need access to `self` to issue a warning message that's specific to the instance.